src/docs/meta/relay/best-practices.mdx

```diff
@@ -46,21 +108,15 @@ limits:
   max_thread_count: 8
   # Number of concurrent upstream requests for event submission
   max_concurrent_requests: 500
```

Comment:
@jan-auer Do you think we should `max_concurrent_requests` too?

Reply:
Do you mean `max_concurrent_queries`? I'd say no, since Relays will only run for a single organization, so the number of projects will stay in reasonable bounds. This is in contrast to the event volume, which varies greatly from organization to organization, so it is worth thinking about this.

Reply:
What I meant to ask was, do we want to keep or remove `max_concurrent_requests` in this example?
I think we should remove it: because if we encourage to tweak it, what stops users from setting it to 100000 to get it out of the way

Reply:
Good point, this makes me think that we need to limit this anyway. I still think it makes sense to leave it there, but I'm following your lead here.